x/observer/migrations/v11/migrate_test.go

```diff
@@ -0,0 +1,57 @@
+package v11_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/zeta-chain/node/pkg/chains"
+	keepertest "github.com/zeta-chain/node/testutil/keeper"
+	"github.com/zeta-chain/node/testutil/sample"
+	v11 "github.com/zeta-chain/node/x/observer/migrations/v11"
+	"github.com/zeta-chain/node/x/observer/types"
+)
+
+// getTestChainParams returns a list of chain params for testing
+func getTestChainParams() types.ChainParamsList {
+	return types.ChainParamsList{
+		ChainParams: []*types.ChainParams{
+			sample.ChainParams(chains.Ethereum.ChainId),
+			sample.ChainParams(chains.BscMainnet.ChainId),
+			sample.ChainParams(chains.Amoy.ChainId),
+			sample.ChainParams(chains.ArbitrumMainnet.ChainId),
+		},
+	}
+}
+
+func TestMigrateStore(t *testing.T) {
+	t.Run("can migrate chain params", func(t *testing.T) {
+		k, ctx, _, _ := keepertest.ObserverKeeper(t)
+
+		// set chain params
+		testChainParams := getTestChainParams()
+		k.SetChainParamsList(ctx, testChainParams)
+
+		// ensure the chain params are set correctly
+		oldChainParams, found := k.GetChainParamsList(ctx)
+		require.True(t, found)
+		require.Equal(t, testChainParams, oldChainParams)
+
+		// migrate the store
+		err := v11.MigrateStore(ctx, *k)
+		require.NoError(t, err)
+
+		// ensure we still have same number of chain params after migration
+		newChainParams, found := k.GetChainParamsList(ctx)
+		require.True(t, found)
+		require.Len(t, newChainParams.ChainParams, len(oldChainParams.ChainParams))
+
+		for _, params := range newChainParams.ChainParams {
```

Comment:
NIT: Maybe add a check to make other values are identical

Reply:
bc50d5896